### PR TITLE
Middle sections (Works and About me)

### PR DESCRIPTION
**Pull request by teckim**
in this pull request:

- Created Work section.
- Created Work card component.
- Improved components.
- Added About me section.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -65,5 +70,214 @@ <h1 class="headline__title">
...
+          <div class="about-me__text">
+            <p>
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+          </div>
+          <div class="about-me__actions">
+            <a class="link" href="#">LET’S CONNECT</a>
```
Comment (zhadier) [inline src/index.html]: - [x] I think we should make this `<a>` tag a `<h3>` or `<h4>` tag. It's not actually a link so it's kind of misleading and will cause problems for screen readers.

File: src/index.html [hunk 2]
```diff
@@ -65,5 +70,214 @@ <h1 class="headline__title">
...
+      </div>
+      <div class="project-card">
+        <img
+          class="project-card__image"
+          src="./assets/images/snapshoot-2.jpg"
+          alt="Snapshoot 1"
+        />
+        <div class="project-card__body">
+          <div class="project-card__header">
+            <h2 class="project-card__header-title">Multi-PostStories</h2>
```
Comment (zhadier) [inline src/index.html]: - [x] [Optional] Just a small typo here in Multi-Post Stories. I saw it when I was looking at your design, you can fix it whenever feasible :)

File: src/index.html [hunk 3]
```diff
@@ -65,5 +70,214 @@ <h1 class="headline__title">
...
+              </li>
+              <li>
+                <a class="icon-botton">
+                  <img src="./assets/icons/linkedin.svg" alt="" />
+                </a>
+              </li>
+              <li>
+                <a class="icon-botton">
+                  <img src="./assets/icons/twitter.svg" alt="" />
+                </a>
```
Comment (zhadier) [inline src/index.html]: - [x] What do you think about adding the AngelList and Medium icons here as well? I know you might not be having accounts for them right now but in the near future, Microverse will be helping you create accounts for both of them and they're both popular platforms. It's better to have the icons right now so you would not have to change your Portfolio in the near future :)

File: src/css/elements/button.css [hunk 4]
```diff
@@ -0,0 +1,18 @@
+.button {
+  font-weight: var(--font-medium);
+  font-size: var(--text-md);
+  color: var(--color-primary-B500);
+  line-height: 24px;
+  padding: 12px;
+  border-radius: 8px;
+  border: 1px solid var(--color-primary-B400);
+}
```
Comment (zhadier) [inline src/css/elements/button.css]: - [x] You seem to have missed implementing button interactions in your Portfolio as was required. You can refer to the image below to see what interactions you have to implement from the Figma template

image

To do so, you can use different CSS selectors on your buttons. For example, you can use the [`:hover`](https://www.w3schools.com/cssref/sel_hover.asp) selector to add hover effects to your elements

**Review by zhadier — CHANGES_REQUESTED** [review]
# Hi @teckim 👋  

Great progress on the Project, your design looks absolutely amazing and your file structure is one of the best I've seen 🔥 🔥   
There are some issues that you still need to work on to go to the next project but you are almost there! 🌞 



## Highlights
✔️ Amazing design and clean code
✔️ Linters setup correctly and working perfectly
✔️ Great PR description and commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @zhadier in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (teckim) [reply src/index.html]: Oh! I thought that it's a link that takes to the contact form.

File: src/css/elements/button.css (hunk 4 above)
Comment (teckim) [reply src/css/elements/button.css]: ### Thank you for your comment.
issue fixed here [Add button variants](https://github.com/teckim/portfolio/pull/2/commits/50e24c255599cebcf41adfea99399f68e449a95e)

File: src/index.html (hunk 3 above)
Comment (teckim) [reply src/index.html]: ### Thank you for your comment.
issue fixed here [Add AngelList and Medium icons](https://github.com/teckim/portfolio/pull/2/commits/2e27d9eebe1fa42076b7340ddd3f25ec4d1a508c)

File: src/index.html (hunk 2 above)
Comment (teckim) [reply src/index.html]: ### Thank you for your comment.
issue fixed here [Fix Typo in Home page](https://github.com/teckim/portfolio/pull/2/commits/e337d177aaac83ff5a8f873a4db8549c05c6aadf)

File: src/index.html (hunk 1 above)
Comment (teckim) [reply src/index.html]: ### Thank you for your comment.
issue fixed here [Fix headline and about subtitles](https://github.com/teckim/portfolio/pull/2/commits/7132c51f661d68ba5ba98574bc9b248b8feedd5e)

**Review by danmainah — APPROVED** [review]
Hi @teckim , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design
✔️  Good PR and Description

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me using @danmainah in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**